### PR TITLE
fix vertical overflow scroll on `BackButton`

### PR DESCRIPTION
**Pull request by gitstart**
Fixes: https://github.com/chatwoot/chatwoot/issues/6913

## Description

Removes `settings` class name from `BackButton` to prevent vertical scrollbar on Button

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

- Go to Teams
- Click on Create new teams
- Notice there's no scrollbar on the back button

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

## Demo

Screenshot 2023-04-17 at 9 55 50 AM

### CODE REVIEW

**Review by scmmishra — COMMENTED** [review]
@gitstart @raph941 this might break RTL view, can you confirm?

CC: @iamsivin

**Comment by iamsivin** [comment]
> @gitstart @raph941 this might break RTL view, can you confirm?
> 
> CC: @iamsivin

Yes, It will break. I can add a fix for it.

File: app/javascript/dashboard/components/widgets/BackButton.vue [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <button class="settings back-button" @click.capture="goBack">
+  <button class="back-button" @click.capture="goBack">
```
Comment (iamsivin) [inline app/javascript/dashboard/components/widgets/BackButton.vue:2]: @gitstart Can you give permission to push the changes, or can you make changes here?
There is an SCSS file called `_rtl.scss` [line](https://github.com/chatwoot/chatwoot/blob/76d4c22c2d22f2c88ce59bec6611d016360456a2/app/javascript/dashboard/assets/scss/_rtl.scss#L88) 88 , change the class name to `.header-section.back-button`

```suggestion
  <button class="header-section back-button" @click.capture="goBack">
```

Comment (gitstart) [reply app/javascript/dashboard/components/widgets/BackButton.vue]: Hi @iamsivin, sorry for the delay. 
Thanks for the fix, the PR has now been updated.

**Review by nithindavid — APPROVED** [review]

**Comment by nithindavid** [comment]
@ raph941 Thanks for the PR :)

**Pull request merged**